ocs_ci/ocs/platform_nodes.py

```diff
@@ -410,6 +438,19 @@ def create_and_attach_nodes_to_cluster(self, node_conf, node_type, num_nodes):
         node_cls_obj = node_cls(node_conf, node_type, num_nodes)
         node_cls_obj.add_node()
 
+    def terminate_nodes(self, nodes, wait=True):
+        """
+        Terminate the VM's
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            wait (bool): True for waiting the instances to terminate,
+            False otherwise
+
+        """
+        vms = self.get_vm_nodes_in_dc(nodes)
+        self.vsphere.destroy_vms(vms)
```

Comment:
will it also delete the attached data disk to the VM if any? or just remove the VM from inventory

Reply:
From what I know, it should delete the attached data disk to the VM.